extensions/wikia/CommunityPage/styles/components/_header.scss

```diff
@@ -40,7 +40,7 @@
 
 .community-page-admin-welcome-message {
 	align-items: center;
-	background-color: $color-community-page-gray;
+	background-color: $color-unified-module-background;
 	color: $color-text-dark;
```

Comment:
`background-color` is calculated based on wiki theme so text with `$color-text-dark` may not has enough contrast.